apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -42,6 +44,29 @@ function showHomepage() {
     store.dispatch(setMapboxAccessToken(homepageData.mapboxAccessToken));
   }
 
+  // remove courseOfferingId, courseVersionId, and unitId params so that if we
+  // navigate back we don't get the create section dialog again
```

Comment:
Good thinking, and thank you for adding a comment to explain

Reply:
I have Erin Bond to thank for this :)